connect.yaml

```diff
@@ -21,7 +21,7 @@ deployAs:
           description: Session API URL
           required: true
           default: https://session.europe-west1.gcp.commercetools.com
-        - key: CTP_CLIENT_ID
+        - key: CTP_CLIENT_ID  with manage_payments, manage_orders, view_sessions, view_api_clients, manage_checkout_payment_intents & introspect_oauth_tokens scopes
```

Comment:
@prateek-ct @johnsonogwuru this text should go in the description and not in the key. Can you please fix it?

Reply:
https://github.com/commercetools/connect-payment-integration-adyen/pull/140